working/macros/feature-specification.md

```diff
@@ -896,6 +916,28 @@ It is a compile-time error if an macro class constructor invoked by a macro
 application has a parameter whose type is not Code (or any subtype of it) or
 one of the aforementioned primitive types (or a nullable type of any of those).
 
+#### Identifier Scope
+
+The following rules apply to any `Identifier` passed as an argument to a macro
+application, whether as a part of a `Code` expression or directly as an
+`Identifier` instance.
+
+The scope of any `Identifier` argument is the same as the scope in which the
+identifier appears in the source code, which is the same as the scope for any
+other type of annotation on a declaration. This means:
```

Comment:
"for any other type of annotation" -> "argument scope for a metadata annotation".